fix(Input): change action subscribe event to onChange - fixes #47

SamuelACook opened the pull request:
The onAxis event does not emit changes from axis data when the axis
is reset to zero. Switching to the onChange event will ensure even if
the axis data changes to zero then the event is still emitted.
resolves #47

thestonefox commented:
Can you amend your commit message to ensure it is no longer than 72 characters per line, the scope is `Input` and it has no mention of the github issue in it (this should be added after in github and not appear in the git history)

so the commit message would read like:

```
fix(Input): change action subscribe event to onChange

The onAxis event does not emit changes from axis data when the axis
is reset to zero. Switching to the onChange event will ensure even if
the axis data changes to zero then the event is still emitted.
```

SamuelACook commented:
Okay, I've updated the commit message to reads as mentioned above. I've also updated the title and comment to reflect the commit message change. For the pull request, I kept the appended "resolve issue" in the Github comment. Let me know if you don't want that in the pull request comments.

thestonefox commented:
All looks good now, fine keeping anything in the github comments.
